weblate/addons/models.py

```diff
@@ -208,34 +209,37 @@ def handle_addon_error(addon, component):
         addon.disable()
 
 
-@receiver(vcs_pre_push)
-def pre_push(sender, component, **kwargs):
-    for addon in Addon.objects.filter_event(component, EVENT_PRE_PUSH):
-        component.log_debug("running pre_push add-on: %s", addon.name)
+def handle_addon_event(sender, component, event_type, **kwargs):
+    """Executing addon event for a particular event type."""
+    event_string = EVENT_STRING.get(event_type)
+    translation = kwargs.get("translation")
+    for addon in Addon.objects.filter_event(component, event_type):
+        translation.log_debug(f"running {event_string} add-on: {addon.name}")
```

Comment:
This might be `None` because of the above `kwargs.get` and that would crash here.

Reply:
Sounds right